src/Controls/src/SourceGen/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "profiles": {
```

Comment:
Is this something we want to check in ?

Reply:
I actually think we do. It provides a simple debug target to debug the source generator.